grip-mode.el

```diff
@@ -98,6 +103,16 @@ option."
 (defvar-local grip--preview-file nil
   "The preview file for grip process.")
 
+(defun grip--browser (url)
+  "Use browser specified by user to load URL.
+Use default browser if nil."
+  (if grip-url-browser
+      (progn
+       (setq-local browse-url-generic-program grip-url-browser
```

Comment:
Doesn't `let` work here? I prefer `let` than `setq-local`.

Or use this
```elisp
(when grip-url-browser
  (setq-local browse-url-generic-program grip-url-browser
              browse-url-browser-function 'browse-url-generic))
(browse-url url)
```

Reply:
> let


For some reason let is causing issues with void function. **when** works as expected

Reply:
OK.